environments/datastore/action-hooks/package.json

```diff
@@ -3,6 +3,7 @@
   "name": "@aws-amplify/ui-action-hooks-environment",
   "version": "0.0.1",
   "scripts": {
-    "pull": "amplify pull --appId d1c6vwjkmuz6ml --envName dev"
+    "pull": "amplify pull --appId d1c6vwjkmuz6ml --envName dev",
+    "generate": "echo 'Not available yet. Generating a placeholder file for now...' && touch amplify_outputs.json && echo '{}' > amplify_outputs.json"
```

Comment:
Let's rethink these "Not available yet" messages, "yet" creates an assumption that support is on the way which may not be true, but even if there is support coming that allows this message to be removed in the future, how do readers correlate that support to this script?

Reply:
I will update those to the real commands. When I opened this PR, I did not finish the creation for some of the components 😅